hypershift-operator/controllers/machineconfigserver/manifests.go

```diff
@@ -55,11 +56,11 @@ func MachineConfigServerIgnitionRoute(machineConfigServerNamespace, machineConfi
 	}
 }
 
-func MachineConfigServerUserDataSecret(mco *hyperv1.MachineConfigServer) *corev1.Secret {
+func MachineConfigServerUserDataSecret(mcs *hyperv1.MachineConfigServer) *corev1.Secret {
 	return &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
-			Namespace: mco.GetNamespace(),
-			Name:      fmt.Sprintf("user-data-%s", mco.GetName()),
```

Comment:
Any specific reason for changing the secret name? The new name no longer suggests much about what the secret might be used for (not that it matters much, unless there comes a time we need two secrets for the MCS...)

Reply:
we might need two at some point, but for such a case we'd probably want to append the version or similar. I'll put it back as you mentioned to clarify its purpose